src/View/ThemeResourceLoader.php

```diff
@@ -328,6 +328,9 @@ public function findThemedResource($resource, $themes = null)
         foreach ($paths as $themePath) {
             $relativePath = Path::join($themePath, $resource);
             $absolutePath = Path::join($this->base, $relativePath);
+            if (!Path::isBasePath($themePath, $relativePath) || !Path::isBasePath($this->base, $absolutePath)) {
+                continue;
+            }
```

Comment:
Protecting against path traversal here just to avoid accidental modification of files that shouldn't be modified.
Not throwing an exception because this is consistent with what happens if the file just doesn't exist anyway.